### PR TITLE
Doorway Prefab

### PR DESCRIPTION
**Pull request by fnnbrr**
No messing with code here - just a door model, a lamp model, and a "doorway" prefab that can be dragged right into the MAIN scene where needed.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -75,3 +75,6 @@ fmod_editor.log
 /FMOD/**.cache
 /FMOD/**.restore
 game-jam-2.sln.DotSettings
+ProjectSettings/QualitySettings.asset
+ProjectSettings/Packages/com.unity.probuilder/Settings.json
```
Comment (jurcaua) [inline .gitignore]: As much as I hate it, I wouldn't push this because if we ever actually update the settings, we'll want to make sure everyone gets them

**Review by jurcaua — CHANGES_REQUESTED** [review]
I wouldnt update the .gitignore, but good to push otherwise!

File: .gitignore (hunk 1 above)
Comment (fnnbrr) [reply .gitignore]: Gotcha, fixed

**Review by jurcaua — APPROVED** [review]
I love it so much
MERGE IT UP

**Pull request merged**